### PR TITLE
Feature/xinnanli/localize settings page

### PR DESCRIPTION
**Pull request by xinnanlisf**
# Critical Changes

# Changes
Replaced all labels in HEDA Settings with custom labels to facilitate language translations. 
# Issues Closed

### CODE REVIEW

**Comment by xinnanlisf** [comment]
@smarovich 
Hi, Sarah, would you mind reviewing this pull request, please? Thanks!

**Comment by lianeyee** [comment]
@xinnanlisf I'm tempted to say something like "Replaced all labels in HEDA Settings with custom labels to facilitate language translations." However, I'm not sure that it's accurate to say "all labels." Are there any exceptions for certain strings?

**Comment by xinnanlisf** [comment]
@lianeyee 
Sorry for the confusions. No, we are replacing all the hard coded strings in the settings page to support future implementation of translations.

And I think your release note looks great!

Let me know if it makes sense! :D

**Comment by lianeyee** [comment]
Thanks for confirming, @xinnanlisf. You got us started with a great draft note. I've updated Changes.

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -329,18 +665,26 @@
...
+        <protected>true</protected>
+        <shortDescription>stgHelpAccoutsDeletedIfChildContactsDeleted</shortDescription>
+        <value>Record Types of those Accounts that will be automatically deleted if all their children Contacts have been deleted.</value>
+    </labels>
+    <labels>
         <fullName>stgHelpAddressAccRecType</fullName>
         <categories>Settings</categories>
         <language>en_US</language>
-        <protected>false</protected>
+        <protected>true</protected>
```
Comment (davidjray) [inline src/labels/CustomLabels.labels]: I am curious, why are the labels being changed to protected?

Comment (xinnanlisf) [reply src/labels/CustomLabels.labels]: @davidjray 
Basically I just manually changed them in the xml file. Looks like I accidentally changed existing custom labels as well. Should I change them back?

Comment (davidjray) [reply src/labels/CustomLabels.labels]: I'm not sure. I think we should have a standard we follow. @ceiroa what are your thoughts?

**Comment by ceiroa** [comment]
New labels should be created protected so we can delete them later if we want to. If they aren't protected they cannot ever be removed from the package, as there might be code in the org that references them. Unfortunately the packaging org won't let you change a non-protected label to protected, for the same reason.

File: src/labels/CustomLabels.labels [hunk 2]
```diff
@@ -284,35 +580,67 @@
         <fullName>stgDefaultFacultyTypeTitle</fullName>
         <categories>Settings</categories>
         <language>en_US</language>
-        <protected>false</protected>
```
Comment (davidjray) [inline src/labels/CustomLabels.labels]: I agree with @ceiroa, that it is a good idea to make all new labels protected. Unfortunately, we cannot change existing labels due to packaging restrictions. So we'll need to change these back.

Comment (xinnanlisf) [reply src/labels/CustomLabels.labels]: @davidjray 

I see I will change those back, thanks!

**Comment by xinnanlisf** [comment]
@davidjray should be good now!

fyi, @ceiroa

**Comment by davidjray** [comment]
@sreesureka and @chrismcmahon this PR is ready for QA!

**Comment by davidjray** [comment]
@chrismcmahon I see you were assigned. The best way to test this is to complete some exploratory testing in the HEDA settings page. Visiting each tab and making some arbitrary changes.

**Comment by chrismcmahon** [comment]
Checked HEDA Settings, all the text looks correct and Settings save properly

**Pull request merged**